test/e2e/e2e-simple.sh

```diff
@@ -2,8 +2,10 @@
 
 set -eu
 
-source test/lib.sh
-source test/testcases.sh
+source test/e2e/lib/check.sh
+source test/e2e/lib/workflow.sh
+source test/e2e/lib/util.sh
+source test/e2e/testcases.sh
```

Comment:
I know that when run from the Makefile target these will source fine, but from any other directory they won't. Any chance you could do something like:

```shell
SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
source "$SCRIPT_DIR/lib/check.sh"
source "$SCRIPT_DIR/lib/workflow.sh"
source "$SCRIPT_DIR/lib/util.sh"
source "$SCRIPT_DIR/testcases.sh"
```

Reply:
Will do. Thanks!